src/main/java/datastax/astra/migrate/AbstractJobSession.java

```diff
@@ -42,7 +42,7 @@ protected AbstractJobSession(CqlSession sourceSession, CqlSession astraSession,
 
         readLimiter = RateLimiter.create(new Integer(Util.getSparkPropOr(sc, "spark.readRateLimit", "20000")));
         writeLimiter = RateLimiter.create(new Integer(Util.getSparkPropOr(sc, "spark.writeRateLimit", "40000")));
-        maxRetries = Integer.parseInt(sc.get("spark.maxRetries", "10"));
+        maxRetries = Integer.parseInt(sc.get("spark.maxRetries", "0"));
```

Comment:
curious, why are we removing the default retry mechanism? Is it taking too long?

Reply:
@msmygit Based on experience, retries rarely help & when it does, we usually end up reporting incorrect stats. The stats issue is difficult to address as when a particular partition-range fails, we do not know how many records it actually migrated before failing during each attempt. We will need complex logic to track this for every attempt.

Earlier we preferred retires as we did not have an option to rerun only the failed partitions & had to rerun the full migration again, but we added feature to migrate only failed partitions https://github.com/datastax/cassandra-data-migrator#migrating-specific-partition-ranges & that way we always report accurate stats.